contracts/middle-layer/TokenHub.sol

```diff
@@ -62,6 +62,11 @@ contract TokenHub is Config, ReentrancyGuardUpgradeable, IMiddleLayer {
         _;
     }
 
+    modifier transferAmountCheck(uint256 _amount) {
```

Comment:
I think we still allow transfer larger than 1000BNB, but just time lock it if it exceed this.

Reply:
fixed